.travis.yml

```diff
@@ -12,10 +12,16 @@ env:
 before_install:
 - gem install bundler
 before_script:
+- cp ./v2_key.dev ./v2_key
 - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
   > ./cc-test-reporter
 - chmod +x ./cc-test-reporter
 - "./cc-test-reporter before-build"
-- bundle exec rake db:create db:migrate
+- bundle install
+- cd spec/dummy && bin/setup && cd -
+- bundle exec rake db:create
```

Comment:
Are these necessary if `spec/dummy/bin/setup` is already doing them?

Reply:
👍